tools/vuln_valid/index.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+const Joi = require('joi');
+const Path = require('path');
+const Fs = require('fs');
+
+const vulnPath = Path.join(__dirname, '..', '..', 'vuln');
+const coreVulnPath = Path.join(vulnPath, 'core');
+const npmVulnPath = Path.join(vulnPath, 'npm');
+
+const coreModel = Joi.object().keys({
+    cve: Joi.array().items(Joi.string().regex(/CVE-\d{4}-\d+/)),
```

Comment:
Can you add explicit `required()` and `optional()` calls to the schema. Right now you have some `optional()`s, but I believe that is the default anyway, so the intentions are not really clear.